Documentation/Cassette/README.md

```diff
@@ -8,16 +8,23 @@
 | 1   | mg 996r servo         |
 
 ## Respooler
-| Qty | Item                  |
-|-----|-----------------------|
-| 4   | 608 bearings          |
-| 2   | 80mmx8mm shaft        |
-| 1   | 8x12mm one way bearing|
-| 2   | #110 Orings 13/16" ID 1 1/16" OD  |
+| Qty | Item                                                                                           |
+|-----|------------------------------------------------------------------------------------------------|
+| 5   | 608 bearings                                                                                   |
+| 2   | 80mmx8mm shaft                                                                                 |
+| 1   | HF081412 one way bearing (8mm ID, 14mm OD, 12mm L)                                             |
+| 2   | #211 Orings (13/16" ID, 1 1/16" OD)                                                            |
+| 8   | M3 Heatset inserts (5mm OD, 4mm L)                                                             |
+| 1   | ECAS04 (remove rubber spacer)                                                                  |
+| 1   | Spring (6mm OD, 1mm Wire Size, 7.5mm Compressed Length, 15mm Free Length, 37.2N Load Capacity) |
+| 2   | M3x25mm Socket head cap screw                                                                  |
```

Comment:
1 for the the tension mechanism (i.e. through the spring)...though actual length can vary.
1 for the tension arm pivot point. Not sure if this was intended to be an m3 shaft? I just cut threads into plastic and let the screw protrude (seen on left side of the picture of the tension arm). Note: outer dimension of the tension arm looks to be 24mm. So perhaps an m3 x 24mm shaft/pin is intended here?